feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenUI.kt

```diff
@@ -200,15 +208,16 @@ private fun WillFeedConfirmationDialog(
     bottomSheetState: AnimealBottomSheetState,
     state: HomeState,
     onScreenEvent: (HomeScreenEvent) -> Unit,
-    onHideBottomSheet: () -> Unit
+    onWillFeedEvent: (WillFeedEvent) -> Unit,
+    onHideBottomSheet: () -> Unit,
 ) {
     if (state.willFeedState is WillFeedState.Showing) {
```

Comment:
Can we move this if into dialog in feeding?

Reply:
I meant into `feeding` feature in the body of `FeedConfirmationDialog`. 
Because we repeat this if statement in 3 features, when we can just pass `WillFeedState` to dialog.

Reply:
Done! sorry didn't get it 🤣